packages/catalog-process/src/services/catalogService.ts

```diff
@@ -981,12 +984,33 @@ export async function cloneDescriptorLogic({
     name: string
   ) => Promise<string>;
   eService: WithMetadata<EService> | undefined;
+  getEServiceByNameAndProducerId: ({
+    name,
+    producerId,
+  }: {
+    name: string;
+    producerId: string;
+  }) => Promise<WithMetadata<EService> | undefined>;
 }): Promise<{ eService: EService; event: CreateEvent<EServiceEvent> }> {
   assertEServiceExist(eServiceId, eService);
   assertRequesterAllowed(eService.data.producerId, authData.organizationId);
 
-  const descriptor = retrieveDescriptor(descriptorId, eService);
+  const currentDate = new Date();
+  const formattedDateAndTime = `${currentDate.toLocaleDateString(
+    "it-IT"
+  )} ${currentDate.toLocaleTimeString("it-IT")}`;
```

Comment:
Interesting. Just a note: I wrote the code in that way because the desired suffix (as specified in the ticket by @beetlecrunch) is `- clone - dd/MM/yy HH:mm:ss` and also in Scala it has been implemented without the comma. (https://github.com/pagopa/interop-be-catalog-management/pull/235)